src/libraries/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.Initialization.cs

```diff
@@ -42,13 +76,195 @@ static CryptoInitializer()
                 throw new InvalidOperationException();
             }
         }
+#pragma warning restore CA1810
 
         internal static void Initialize()
         {
             // No-op that exists to provide a hook for other static constructors.
         }
 
+        private static bool GetMemoryDebug(string name)
+        {
+            string? value = Environment.GetEnvironmentVariable(name);
+            if (int.TryParse(value, CultureInfo.InvariantCulture, out int enabled))
+            {
+                return enabled == 1;
+            }
+
+            return false;
+        }
+
+        internal static void EnableTracking()
+        {
+            _allocationsDiff!.Clear();
+            _trackIncrementalAllocations = true;
+        }
```

Comment:
This will throw NRE if the required environment variables were not set beforehand. Perhaps a better exception  would be useful?